src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -148,7 +150,10 @@ export const NewProposal = ({ navigation, data, query }) => {
 
     setIsLoading(true);
 
-    if (newProposalData.image) {
+    // if the image is an absolute url, we are editing and do not want to upload a new image
+    const isImageToUpload = !URL_REGEX.test(newProposalData.image);
+
+    if (newProposalData.image && isImageToUpload) {
```

Comment:
i guess we should go even further in refactoring and do

```
const isImageToUpload = newProposalData.image && !URL_REGEX.test(newProposalData.image);

if (isImageToUpload) {
```

otherwise `isImageToUpload` could be `true` but there is no image, just because the regex test returns `false`. this would contradict the variable name.

Reply:
[2d81526](https://github.com/ikuseiGmbH/smart-village-app-app/pull/539/commits/2d81526eb0d4553070cace89fba45903e64c0360)